server/src/service/CardService.js

```diff
@@ -185,4 +186,50 @@ export class CardService extends BaseService {
             })),
         };
     }
+
+    @Transactional()
+    async createCard({ userId, listId, title, dueDate, content }) {
+        const list = await this.listRepository
+            .createQueryBuilder('list')
+            .select(['list.id', 'list.board'])
+            .where('list.id = :listId', { listId })
+            .getRawOne();
+        if (!list) throw new EntityNotFoundError();
+        const boardService = BoardService.getInstance();
+        await boardService.checkForbidden(userId, list.board_id);
+
+        const cardWithMaxPosition = await this.cardRepository
+            .createQueryBuilder('card')
+            .select('MAX(card.position)', 'maxPosition')
+            .where('card.list = :listId', { listId })
+            .getRawOne();
+        const card = {
+            title,
+            content,
+            position: cardWithMaxPosition.maxPosition + 1,
+            dueDate,
+            list: list.list_id,
+            creator: userId,
+        };
+        const createCard = this.cardRepository.create(card);
+        await this.cardRepository.save(createCard);
+    }
+
+    @Transactional()
+    async deleteCard({ userId, cardId }) {
+        const card = await this.cardRepository
+            .createQueryBuilder('card')
+            .select(['card.id', 'card.list'])
+            .where('card.id = :cardId', { cardId })
+            .getRawOne();
```

Comment:
```suggestion
        const card = await this.cardRepository
            .findOne(cardId, {
                loadRelationIds: {
                    relations: ['list'],
                    disableMixedMap: true,
                },
            })
```
위와 같이 하시면, 조인하지 않고 card.list.id에 값을 넣어줍니당

Reply:
오 이렇게도 되는군요 !
한가지 여쭤보고 싶은게 typeorm에서 쿼리빌더와 find메서드의 성능차이가 있나요?

Reply:
아... card.list로 가져오시면 조인을 하지 않겠네요. 조인을 하지 않는다면 성능차이는 없겠네요!